icaruscode/CRT/CRTDataAnalysis_module.cc

```diff
@@ -412,7 +413,7 @@ namespace crt {
 	for(int chan=0; chan<32; chan++) {
 	  std::pair<double,double> const chg_cal = fChannelMap->getSideCRTCalibrationMap((int)crtList[febdat_i]->fMac5,chan);
 	  float pe = (crtList[febdat_i]->fAdc[chan]-chg_cal.second)/chg_cal.first;
-	  if(pe<=fPEThresh) continue;
+	  if(pe<=fPEThresh && !crtList[febdat_i]->IsReference_TS1()) continue;
```

Comment:
I do confirm this is intended. It could also be that in future IsReference_TS0() will be added. These reset events are used for improvements for timing, they get filtered out in the HitReco stage.